src/compiler.ts

```diff
@@ -240,7 +240,7 @@ export class Options {
   /** Global aliases, mapping alias names as the key to internal names to be aliased as the value. */
   globalAliases: Map<string,string> | null = null;
   /** Features to activate by default. These are the finished proposals. */
-  features: Feature = Feature.MUTABLE_GLOBALS;
+  features: Feature = Feature.MUTABLE_GLOBALS | Feature.SIGN_EXTENSION;
```

Comment:
This change also needs to be reflected in `cli/asc.json` I think, which currently mentions it as disabled by default, but now needs to move it, pointing out that it can be disabled with `--disable`.